app/utils/ajax-promise.js

```diff
@@ -1,27 +1,37 @@
 import Ember from 'ember';
 import config from '../config/environment';
 
-export default function(url, type, authToken, data, args) {
-  return new Ember.RSVP.Promise(function(resolve, reject) {
+export default function (url, type, authToken, data, args, language = null) {
```

Comment:
Can we have default 'en' instead of null??

Reply:
Yup.. Doable and makes sense. Thanks :+1: